src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
As it stands currently, if someone visited a web page on Chrome for Android that was using traditional Web Notifications, would this not cause a case positive and result in an error? It is breakages and false positives like this we need to be careful to avoid.